### PR TITLE
Check field nationality

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: src/Service/HaalCentraalToStufBGService.php [hunk 1]
```diff
@@ -135,6 +135,31 @@ public function getAllRelatives(Source $source, array $ingeschrevenPersoon): ?ar
...
+     * @return array|null If the nationality of a ingeschreven persoon is Dutch.
+     */
+    public function checkDutchNationality(array $ingeschrevenPersoon): ?array
+    {
+        // Set the nationality default on false.
+        $dutchNationality = "false";
+        foreach ($ingeschrevenPersoon['nationaliteiten'] as $nationality) {
+            // If the omschrijving is 'Nederlandse' or the code is '0001' then set the bool to true.
+            if ($nationality['nationaliteit']['omschrijving'] === 'Nederlandse'
+                || $nationality['nationaliteit']['code'] === '0001'
```
Comment (WilcoLouwerse) [inline src/Service/HaalCentraalToStufBGService.php:152]: Did you check if this code will always be 0001? 
(this is the case, probably, but if we are unsure only checking Nederlandse might be better)

**Review by WilcoLouwerse — APPROVED** [review]
Looks good 👍 but check my comment

File: src/Service/HaalCentraalToStufBGService.php (hunk 1 above)
Comment (smisidjan) [reply src/Service/HaalCentraalToStufBGService.php:152]: This is what I could find about the code, maybe we also have to use the 0002? https://brp-api.github.io/Haal-Centraal-BRP-bevragen/features/test-report-persoon-nationaliteit-gba.html

image

Comment (WilcoLouwerse) [reply src/Service/HaalCentraalToStufBGService.php:152]: Check, 👍

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**